lib/iex/lib/iex/pry.ex

```diff
@@ -116,6 +116,7 @@ defmodule IEx.Pry do
 
   The actual line is especially formatted in bold.
   """
+  @spec whereami(Path.t(), non_neg_integer(), pos_integer()) :: {:ok, IO.chardata()} | :error
```

Comment:
We shouldn't really use `Path.t()` here since it's a bit of a lie. `Path.t()` is chardata, but we have a guard to check that this is a strict binary.

```suggestion
  @spec whereami(String.t(), non_neg_integer(), pos_integer()) :: {:ok, IO.chardata()} | :error
```